app/scripts/metamask-controller.js

```diff
@@ -1627,6 +1635,18 @@ module.exports = class MetamaskController extends EventEmitter {
   // MISCELLANEOUS
   //=============================================================================
 
+  getOpenMetaMaskTabs () {
```

Comment:
These wrappers around the `this._get*` methods are almost definitely unnecessary

Reply:
d5e881528